python/ray/tests/test_metrics_agent.py

```diff
@@ -59,6 +59,12 @@
     "ray_operation_run_time_ms",
     "ray_operation_queue_time_ms",
     "ray_operation_active_count",
+    # "ray_placement_group_creation_latency_ms",
+    # "ray_placement_group_scheduling_latency_ms",
```

Comment:
should we uncomment them?

Reply:
This requires me to create a pg to export, so I commented it out... maybe I can create some pgs to make it work?